packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -243,15 +243,15 @@ export const addAssetToVault = async (
   const vaultFactoryCreator = E.get(vaultFactoryKit).creatorFacet;
   await E(vaultFactoryCreator).addVaultType(interchainIssuer, oracleBrand, {
     debtLimit: AmountMath.make(stable, BigInt(debtLimitValue)),
-    interestRate: makeRatio(interestRateValue, stable),
+    interestRate: makeRatio(BigInt(interestRateBP), stable, 100_00n),
```

Comment:
Is this supposed to be 10_000 or 100_000?

```suggestion
    interestRate: makeRatio(BigInt(interestRateBP), stable, 10_000n),
```

Reply:
`10_000n` as it is for `liquidationPenalty` a few lines below.

Reply:
It's supposed to be 100n * 100n
I guess it's a goofy way to write it.

Reply:
I might have done that, because BPs are 100ths of a cent. But better to have used a constant multiplier.